docs/source/index.rst

```diff
@@ -2,11 +2,27 @@
 .. image:: svg/blaze_med.png
    :align: center
 
-**Blaze** translates a subset of modified NumPy and Pandas-like syntax to
+Blaze
+-----
+
+**Blaze** is a tool that allows Python users a familiar interface to query data living in other data storage systems. It is the next generation of NumPy, Python’s extremely popular array library. Blaze aims to extend the structural properties of NumPy arrays to a wider variety of table and array-like structures that support commonly requested features such as missing values, type heterogeneity, and labeled arrays. Blaze connects a NumPy/Pandas user interface to external databases.  Users can write intuitive high-level queries in Python and have those queries translated to their database of choice.  This frees the user from the intricacies of their database of choice and lets them move freely between technologies.
```

Comment:
I'd like to avoid the phrase "next generation of NumPy."  This is probably over-reaching.

> that support commonly requested features such as missing values, type heterogeneity, and labeled arrays.

This language is from a few years back and no longer applies to Blaze.  It's more appropriate for something like DyND.